packages/pn-commons/src/services/tracking.service.ts

```diff
@@ -30,16 +30,33 @@ export function trackEvent(event_name: string, nodeEnv: string, properties?: any
 /**
  * Set profile properties
  */
-export function setProfileProperty(property: any, nodeEnv: string): void {
+export function setProfileProperty(
+  propertyType: ProfilePropertyType,
+  property: any,
+  nodeEnv: string
+): void {
   if (!nodeEnv || nodeEnv === 'development') {
     // eslint-disable-next-line no-console
-    console.log('Mixpanel events mock on console log - profile properties');
+    console.log('Mixpanel events mock on console log - profile properties', property);
   } else if (nodeEnv === 'test') {
     return;
   } else {
     try {
       mixpanel.identify(mixpanel.get_distinct_id());
```

Comment:
@alessandrogelmi I don't think is correct to call identify each time we set a profile property, see https://docs.mixpanel.com/docs/tracking-methods/sdks/javascript#identify.
I suggest to move this method call to the mixpanel init function